lib/node_modules/@stdlib/stats/base/dvarmpn/lib/dvarmpn.js

```diff
@@ -43,35 +49,9 @@
 * var v = dvarmpn( x.length, 1.0/3.0, 1, x, 1 );
 * // returns ~4.3333
 */
-function dvarmpn( N, mean, correction, x, stride ) {
-	var ix;
-	var M2;
-	var M;
-	var d;
-	var n;
-	var i;
-
-	n = N - correction;
-	if ( N <= 0 || n <= 0.0 ) {
-		return NaN;
-	}
-	if ( N === 1 || stride === 0 ) {
-		return 0.0;
-	}
-	if ( stride < 0 ) {
-		ix = (1-N) * stride;
-	} else {
-		ix = 0;
-	}
-	M2 = 0.0;
-	M = 0.0;
-	for ( i = 0; i < N; i++ ) {
-		d = x[ ix ] - mean;
-		M2 += d * d;
-		M += d;
-		ix += stride;
-	}
-	return (M2/n) - ((M/N)*(M/n));
+function dvarmpn( N, mean, correction, x, strideX ) {
+	var ox = stride2offset( N, strideX);
+	return ndarray( N, mean, correction, x, strideX, ox );
```

Comment:
```suggestion
	return ndarray( N, mean, correction, x, strideX, stride2offset( N, strideX) );
```
we only define a new variable for this when there are more than 1 offset parameters like in `stats/base/dcumax`. it doesn't really matter very much, just following what we've been doing in other packages here

Reply:
Resolved